generator/win_grpc_generator.cpp

```diff
@@ -159,5 +178,97 @@ grpc::string GetHeaderServices(grpc_generator::File* file){
   return output;
 }
 
+void PrintSourceServerRouterApplyMethodEntry(grpc_generator::Printer* printer,
+                                            const grpc_generator::Method* method,
+                                            std::map<grpc::string, grpc::string>* vars) {
+  if (method->NoStreaming()) {
+    printer->Print(*vars,
+      "if(url == L\"/$Package$$Service$/$Method$\") { \n"
+      "  auto op = std::bind(&Greeter::Service::SayHello, this, " 
+      "  std::placeholders::_1, std::placeholders::_2, std::placeholders::_3);\n"
+      "  boost::wingrpc::handle_request<$Request$, $Response$>(ec, request, response, op); \n"
+      "  return true;\n"
+      "}\n");
+  } else{
+        printer->Print(*vars, "// Streaming for method $Method$ request $Request$ response $Response$ not supported\n");
+  }
+}
+
+void PrintSourceServerRouterApply(grpc_generator::Printer* printer,
+                                  const grpc_generator::Service* service,
+                                  std::map<grpc::string, grpc::string>* vars) {
+  printer->Print(*vars,
+    "bool Greeter::Service::HandleRequest(boost::system::error_code &ec, std::wstring const & url, std::string const & request, std::string & response) {\n");
+  printer->Indent();
+  for (int i = 0; i < service->method_count(); ++i) {
+    PrintSourceServerRouterApplyMethodEntry(printer, service->method(i).get(), vars);
+  }
+  printer->Print(*vars, "return false;\n");
+  printer->Outdent();
+  printer->Print(*vars, "}\n");
+}
+
+void PrintSourceServerMethodSync(grpc_generator::Printer* printer,
+                            const grpc_generator::Method* method,
+                            std::map<grpc::string, grpc::string>* vars) {
+  (*vars)["Method"] = method->name();
```

Comment:
Rmove this function and only retain the variable in vars?

Reply:
I had error after removing this function. Will retain this way